### ISSUE TITLE
[Issue] httpx v1.0.6 Issue with Printing path info to CLI when port is used

### ISSUE THREAD

**Issue by vysecurity** [projectdiscovery/httpx#276]
Current:-

```
echo https://lapp.corp.vipshop.com:8443/core | httpx -status-code -title
https://lapp.corp.vipshop.com [302] [Object moved]
```
I would expect it to show the full URL not just the protocol, domain, and port.

Expected:-

```
echo https://lapp.corp.vipshop.com:8443/core | httpx -status-code -title
https://lapp.corp.vipshop.com:8443/core [302] [Object moved]
```

**Comment by vysecurity** [comment]
Seems it's a confirmed bug: 
![image](https://user-images.githubusercontent.com/3596242/118445186-b0530780-b720-11eb-8cae-4c915e93bd83.png)

**Comment by ehsandeep** [comment]
@vysecurity looks like an issue with printing to CLI, the request is working as intended.

```json
echo https://lapp.corp.vipshop.com:8443/core | httpx -status-code -title -silent -json | jq .

{
  "timestamp": "2021-05-17T16:35:35.102594+05:30",
  "scheme": "https",
  "port": "8443",
  "path": "/core",
  "body-sha256": "c79580e2d6196b4770e20f74a5571374c7eb5b0b669ba50639d2e5f60bcb79e3",
  "header-sha256": "14e85230107f5788892c26d0a8bf35e33fefc490e1c4d256548ab64a10a18ed4",
  "a": [
    ""
  ],
  "url": "https://lapp.corp.vipshop.com:8443/core",
  "location": "/Login.aspx?ReturnUrl=%2fcore",
  "title": "Object moved",
  "webserver": "nginx",
  "content-type": "text/html",
  "method": "GET",
  "content-length": 146,
  "status-code": 302,
  "response-time": "1.686565834s"
}
```

**Comment by ehsandeep** [comment]
This is something that got introduced in v1.0.6 release as detailed here https://github.com/projectdiscovery/httpx/issues/277

**Comment by ehsandeep** [comment]
Closing from here, as this will be handled in https://github.com/projectdiscovery/httpx/issues/277

**Comment by ehsandeep** [comment]
@vysecurity This is now fixed in #282

**Comment by ehsandeep** [comment]
@vysecurity the fix is now live in the latest release of httpx.
